QuizApp/QuizSession/QuizResultView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+struct QuizResultView: View {
+
+    @ObservedObject var viewModel: QuizResultViewModel
+
+    var body: some View {
+        ZStack {
+            LinearGradient
+                .quizAppGradient
+                .ignoresSafeArea()
+            VStack {
+                Spacer()
+                Text("\(viewModel.numberOfCorrectQuestions)/\(viewModel.numberOfQuestions)")
```

Comment:
You can have `result` published var in the view model so you don't have to set this here.